dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -710,14 +721,6 @@ private String getDockerHint(List<Object> hints, String currentDefault) {
      */
     private boolean isWorkflow(String content, Yaml yaml) {
         if (!Strings.isNullOrEmpty(content)) {
-            try {
```

Comment:
Why was this removed? Did the flow change so you know that it's safe?

Reply:
Yes, all of the yaml should be parsed with SafeConstructor prior to this point.